charts/jaeger/values.yaml

```diff
@@ -379,11 +379,11 @@ collector:
       # nodePort:
     otlp:
       grpc:
-        name: grpc-otlp
+        name: otlp-grpc
         # port: 4317
         # nodePort:
       http:
-        name: http-otlp
+        name: otlp-grpc
```

Comment:
```suggestion
        name: otlp-http
```

Reply:
You're right that this was a mistake, but the actual format should have been `http-otlp`.  Otherwise, we'd have caused a regression you attempted to fix in `0.71.5`!